traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/callbacks/WishlistAddCallback.java

```diff
@@ -0,0 +1,6 @@
+package com.guestlogix.travelercorekit.callbacks;
+
+public interface WishlistAddCallback {
+    void onWishlistToggleSuccess();
```

Comment:
language here doesnt make sense....toggle vs. add?

Please pay a bit more attention to these details, raise them if they dont make sense

Reply:
Model after that iOS PR